dev/dependencyList

```diff
@@ -181,6 +183,7 @@ simpleclient_tracer_otel/0.16.0//simpleclient_tracer_otel-0.16.0.jar
 simpleclient_tracer_otel_agent/0.16.0//simpleclient_tracer_otel_agent-0.16.0.jar
 slf4j-api/1.7.36//slf4j-api-1.7.36.jar
 snakeyaml/1.33//snakeyaml-1.33.jar
+snappy-java/1.1.7.3//snappy-java-1.1.7.3.jar
```

Comment:
does it support arm?

Reply:
when introduce new artifacts in release tarball, don't forget to update LICENSE/NOTICE

Reply:
> snappy-java supports Windows, Mac OS X, Linux (x86, x86_64, arm, etc...)

Acorrding to `snappy-java`'s doc here(https://github.com/xerial/snappy-java/blob/master/BUILD.md), arm architecture should be supported.